cypress/e2e/dashboard.cy.js

```diff
@@ -53,7 +53,7 @@ describe('Validate dashboard page', () => {
   cy.get('[data-testid="myDashboardContent-test"]')
```

Comment:
The `cardTitles` should include CDCP and that should be reflected in the description of the test as well

Reply:
Addressed in 38c77eb7a7e332c6db3bde5ccced9f1e71e174b7